### PR TITLE
added unauthenticated ingress

### PR DESCRIPTION
**Pull request by cody85**

### CODE REVIEW

File: charts/unauthenticated-ingress/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: unauthenticated-ingress
+version: 1.0.12
+home: https://github.com/ManagedKube/helm-charts
```
Comment (piratekev) [inline charts/unauthenticated-ingress/Chart.yaml]: should be thebitgram, not managedkube

File: charts/unauthenticated-ingress/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+standard-application
```
Comment (piratekev) [inline charts/unauthenticated-ingress/README.md]: remove all references to `standard-application` and replace with `unauthenticated_ingress`

File: charts/unauthenticated-ingress/README.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+
+## Configuration
+
+The following table lists the configurable parameters of the chart and the default values.
+
+## Chart Values
+
+| Key | Type | Default | Description |
```
Comment (piratekev) [inline charts/unauthenticated-ingress/README.md]: remove this section

File: charts/unauthenticated-ingress/values.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Default values for standard-application.
+
+#
+# ReplicaCount denotes how many pods you want
+#
+replicaCount: 1
+
+nameOverride: ""
+fullnameOverride: ""
+
```
Comment (piratekev) [inline charts/unauthenticated-ingress/values.yaml:5]: remove these unused values

**Review by piratekev — CHANGES_REQUESTED** [review]

File: charts/unauthenticated-ingress/values.yaml (hunk 4 above)
Comment (cody85) [reply charts/unauthenticated-ingress/values.yaml:5]: Removed other unused values but these are used in templates/_helpers.tpl

**Review by piratekev — APPROVED** [review]

File: charts/unauthenticated-ingress/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: unauthenticated-ingress
+version: 1.0.12
```
Comment (piratekev) [inline charts/unauthenticated-ingress/Chart.yaml]: make this 0.0.1

**Pull request merged**